src/openrct2/Editor.h

```diff
@@ -85,6 +85,12 @@ extern "C"
 
     void editor_open_windows_for_current_step();
 
+    /**
```

Comment:
I have been informed that in fact, doc-block to go above the declaration and not the definition. In other words, what you originally did was correct. Sorry about that.

Reply:
Which means that it is correct now right?

Reply:
Er, yes.